### ISSUE TITLE
Why no ApplyPrivateMethodFunc?

### ISSUE THREAD

**Issue by dablelv** [agiledragon/gomonkey#167]
There is ApplyMethodFunc, why no ApplyPrivateMethodFunc?

Because ApplyMethod and ApplyMethodFunc are a pair, So I think ApplyPrivateMethod and ApplyPrivateMethodFunc should be a pair.

**Comment by agiledragon** [comment]
No, they have different underlying implementations. For ex, the implementation of ApplyPrivateMethod relies on customized reflection.
